examples/products/docpairs.js

```diff
@@ -0,0 +1,111 @@
+var fs = require('fs');
+var zip = require('zip-array').zip
```

Comment:
That shorthand I use sometime works here (using *destructuring assignment*, which you should read up on if you don't know it): `const {zip} = require('zip-array');`